lib/tapioca/generator.rb

```diff
@@ -422,20 +427,25 @@ def compile_gem_rbi(gem)
       say("Compiling #{gem_name}, this may take a few seconds... ")
 
       strictness = config.typed_overrides[gem.name] || "true"
-
+      rbi_body_content = compiler.compile(gem)
       content = String.new
       content << rbi_header(
         config.generate_command,
         reason: "types exported from the `#{gem.name}` gem",
         strictness: strictness
       )
-      content << compiler.compile(gem)
 
       FileUtils.mkdir_p(config.outdir)
       filename = config.outpath / gem.rbi_file_name
-      File.write(filename.to_s, content)
 
-      say("Done", :green)
+      if rbi_body_content.strip.empty?
+        content << EMPTY_RBI_COMMENT
+        say("Done (EMPTY output)", :yellow)
```

Comment:
We don't have to shout 😄 
```suggestion
        say("Done (empty output)", :yellow)
```